common/views/components/License/License.js

```diff
@@ -0,0 +1,51 @@
+import getLicenseInfo from '../../../utils/get-license-info';
+import {spacing, font} from '../../../utils/classnames';
+import Icon from '../Icon/Icon';
+
+function renderAboutAttr(subject) {
+  if(subject) {
+    return `about='${subject}`;
+  }
+}
+
+function renderIcons(icons = []) {
+  return icons.map((i) => (
+    <Icon key={i} name={i} extraClasses={['v-align-middle', 'margin-right-s1']} />
+  ));
+};
+
+function renderLicenseLink(url, icons = [], text = '') {
+  if(url) {
```

Comment:
Not sure if we're forcing the space here (and a few other places)? @davidpmccormick ?

Reply:
I thought we were/should, yeah. Eslint `keyword-spacing` rule _should_ pick this up.